raisinbread/instruments/NDB_BVL_Instrument_medical_history.class.inc

```diff
@@ -284,21 +284,18 @@ class NDB_BVL_Instrument_medical_history extends NDB_BVL_Instrument
         $group[] = $this->form->createElement(
```

Comment:
I think these calls should be converted to `createLabel()` instead.

Reply:
Yes, createLabel() is better, but I just don't want to touch too much the existing code.

Reply:
There are lots of call without createLabel in the existing projects, and createLabel actually calls the createElement, so actually there is no big difference.

Reply:
@lingma i understand your concern but if these projects were using createLabel in the first place, then they would not be affected by correcting the createElement code because we would have just fixed createLabel accordingly and projects would have never know about it.

I think it's fair to give projects a full release time to "amend" their instruments and we would then deprecate the old way in loris 24

Reply:
If you read #6457, you can see that the existing projects will have ZERO impact with the PR.